geth/transactions/txqueue_manager.go

```diff
@@ -75,34 +77,38 @@ func (m *Manager) QueueTransaction(tx *common.QueuedTx) error {
 		to = tx.Args.To.Hex()
 	}
 	log.Info("queue a new transaction", "id", tx.ID, "from", tx.Args.From.Hex(), "to", to)
-	err := m.txQueue.Enqueue(tx)
+	if err := m.txQueue.Enqueue(tx); err != nil {
+		return err
+	}
 	if m.notify {
 		NotifyOnEnqueue(tx)
 	}
-	return err
+	return nil
 }
 
 func (m *Manager) txDone(tx *common.QueuedTx, hash gethcommon.Hash, err error) {
 	m.txQueue.Done(tx.ID, hash, err) //nolint: errcheck
```

Comment:
I think we should handle this error because why not?

Reply:
missed this at first, the only error that can be reported is that tx is not found
i think there is only 3-4 cases when this error can be reported:
- tx discarded but then completed
- tx timed out but then completed, or vice versa

and maybe
- discarded and then timed out

i think in all of them it is safe just to exit without notification, will add tomorrow

Reply:
I would at least log a warning message as it's pretty unusual situation. It will help to debug if there will be any issues with the app.